ethereum/stratum/src/main/java/org/hyperledger/besu/ethereum/stratum/StratumServer.java

```diff
@@ -39,20 +44,27 @@
 public class StratumServer implements PoWObserver {
 
   private static final Logger logger = getLogger();
+  private static final UInt256 divisor =
```

Comment:
`ALL_CALS` and more descriptive?  `BIGINT_TWO_E_256`?
(also, it's bigger than a UInt256 so the type is wrong).

Reply:
Not sure I get this. The value is 1 * 10^32, I think I can maybe represent this in a less obtuse way.

Reply:
I had my brain in hex.

Reply:
I was incorrect, it's 2^256.